include/view.h

```diff
@@ -73,6 +73,7 @@ class View {
 	void push_itemview(std::shared_ptr<RssFeed> f,
 		const std::string& guid,
 		const std::string& searchphrase = "");
+	std::shared_ptr<FormAction> push_empty_form();
```

Comment:
I think it's better to rename this to `push_empty_formaction`, to align better with other methods. (I understand it wasn't possible right away, but now that you removed the previous method of the same name, it can be done.)

Reply:
Makes sense, done: https://github.com/newsboat/newsboat/pull/1259/commits/3061f0443ccc14265165f5b4d9ad69b851ae9a68